.github/container/test-pax.sh

```diff
@@ -388,6 +388,9 @@ else
 fi
 
 if [[ ${EVALUATE} -ne 0 ]]; then
+
+  trap "rm -rf ${OUTPUT}/checkpoints" ERR
```

Comment:
Should we cleanup if this runs successfully as well?

I was thinking maybe:

```suggestion
  trap "rm -rf ${OUTPUT}/checkpoints" ERR INT HUP TERM EXIT
```

I think if you add `EXIT`, you shouldn't need the `rm -rf` that appears below

Reply:
Done, thanks!